elytron/subsystem/src/main/java/org/wildfly/extension/elytron/TlsParser.java

```diff
@@ -58,6 +58,13 @@
 import org.jboss.dmr.ModelNode;
 import org.jboss.dmr.Property;
 import org.jboss.staxmapper.XMLExtendedStreamReader;
+import org.wildfly.extension.elytron_common.CertificateAuthorityAccountDefinition;
+import org.wildfly.extension.elytron_common.CertificateAuthorityDefinition;
+import org.wildfly.extension.elytron_common.FileAttributeDefinitions;
+import org.wildfly.extension.elytron_common.FilteringKeyStoreDefinition;
+import org.wildfly.extension.elytron_common.KeyStoreDefinition;
+import org.wildfly.extension.elytron_common.LdapKeyStoreDefinition;
+import org.wildfly.extension.elytron_common.SSLDefinitions;
 
 /**
  * A parser for the TLS related definitions.
```

Comment:
@cam-rod The simplified TLS subsystem configuration should be placed to the `elytron.common` module in the future?